Extensions/Physics2Behavior/Box2D_v2.3.1_min.wasm.js

```diff
@@ -18,7 +18,7 @@ function Fa(b){for(;0<b.length;){var c=b.shift();if("function"==typeof c)c(a);el
 function na(b){if(a.onAbort)a.onAbort(b);b+="";pa(b);qa(b);ua=!0;throw new WebAssembly.RuntimeError("abort("+b+"). Build with -s ASSERTIONS=1 for more info.");}function Pa(b){var c=Qa;return String.prototype.startsWith?c.startsWith(b):0===c.indexOf(b)}function Ra(){return Pa("data:application/octet-stream;base64,")}
 
 // The wasm file path is defined here.
-var Qa="Extensions/Physics2Behavior/Box2D_v2.3.1_min.wasm.wasm";
+var Qa="Box2D_v2.3.1_min.wasm.wasm";
 
 if(!Ra()){var Sa=Qa;Qa=a.locateFile?a.locateFile(Sa,ia):ia+Sa}
```

Comment:
@D8H You can use `locateFile` to avoid having to modify the JS file: https://emscripten.org/docs/porting/files/packaging_files.html#changing-the-data-file-location

Reply:
We actually use this here (for another purpose): https://github.com/4ian/GDevelop/blob/21b525ab989bde34e4f5c2861bd15d962e20a9d6/newIDE/app/src/index.js#L91-L95